app/views/devise/token.json.jbuilder

```diff
@@ -1,4 +1,7 @@
 json.payload do
   json.success true
   json.partial! 'auth.json.jbuilder', resource: @resource
+  json.data do
+    json.created_at @resource.created_at
+  end
```

Comment:
@sojan-official @tejaswinichile I didn't want to update this attribute for all other APIs, that is why this is written like this. Partial has a `data` attribute, it is getting merged without issues.